salt/orch/kubernetes.sls

```diff
@@ -82,7 +84,12 @@ etcd_proxy_setup:
     # at the same time.
     - batch: 3
     - require:
+{% if salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true', fun='caasp_fqdn', tgt_type='compound')|length == 0 %}
```

Comment:
As a minor comment, for the sake of simplicity, maybe you to move these `if` to the top of the file and set a variable like:

```jinja
{%- set bootstrap_complete = (salt.saltutil.runner('mine.get', tgt='P@roles:kube-(master|minion) and G@bootstrap_complete:true', fun='caasp_fqdn', tgt_type='compound')|length == 0) %}
```

(I don't know if Jinja supports this kind of expression evaluations)

and then you could just

```jinja
{% if bootstrap_complete %}
...
```